docs/source/en/model_doc/univnet.md

```diff
@@ -0,0 +1,49 @@
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# UnivNet
+
+## Overview
+
+The UnivNet model was proposed in [UnivNet: A Neural Vocoder with Multi-Resolution Spectrogram Discriminators for High-Fidelity Waveform Generation](https://arxiv.org/abs/2106.07889) by Won Jang, Dan Lim, Jaesam Yoon, Bongwan Kin, and Juntae Kim.
+The UnivNet model is a generative adversarial network (GAN) trained to synthesize high fidelity speech waveforms. Only the generator of the model, which maps a conditioning log-mel spectrogram and optional noise sequence to a speech waveform (e.g. a vocoder), is implemented here.
+
+The abstract from the paper is the following:
+
+*Most neural vocoders employ band-limited mel-spectrograms to generate waveforms. If full-band spectral features are used as the input, the vocoder can be provided with as much acoustic information as possible. However, in some models employing full-band mel-spectrograms, an over-smoothing problem occurs as part of which non-sharp spectrograms are generated. To address this problem, we propose UnivNet, a neural vocoder that synthesizes high-fidelity waveforms in real time. Inspired by works in the field of voice activity detection, we added a multi-resolution spectrogram discriminator that employs multiple linear spectrogram magnitudes computed using various parameter sets. Using full-band mel-spectrograms as input, we expect to generate high-resolution signals by adding a discriminator that employs spectrograms of multiple resolutions as the input. In an evaluation on a dataset containing information on hundreds of speakers, UnivNet obtained the best objective and subjective results among competing models for both seen and unseen speakers. These results, including the best subjective score for text-to-speech, demonstrate the potential for fast adaptation to new speakers without a need for training from scratch.*
+
+Tips:
+
+- The `noise_sequence` argument for [`UnivNetGan.forward`] should be standard Gaussian noise (such as from `torch.randn`) of shape `([batch_size], noise_length, model.config.model_in_channels)`, where `noise_length` should match the length dimension (dimension 1) of the input `spectrogram`. If not supplied, it will be randomly generated; a `torch.Generator` can be supplied to the `generator` argument so that the forward pass can be reproduced.
+- Padding the end of the input audio with zeros can reduce artifacting at the end of the generated sample. See [this issue](https://github.com/seungwonpark/melgan/issues/8) for more details.
+
+This model was contributed by [dg845](https://huggingface.co/dg845).
```

Comment:
It should be usable with the TTS pipeline: see this [snippet](https://github.com/huggingface/transformers/blob/8881f38a4f48d26c7b2dfe2cb892bfdd63a8ce0b/src/transformers/pipelines/text_to_audio.py#L59-L65).

@dg845 , what's the output sampling rate of this vocoder if any ? If there is one, it might be good to add it to the configuration!

Reply:
I believe the output sampling rate will always be the same as the input sampling rate (which is 24 kHz).

Reply:
I have added a usage example to generate a sample `.wav` file using UnivNet.